### PR TITLE
bugfixing: fix bug in mystudy module-content

### PR DESCRIPTION
**Pull request by nandi-ir**
- add breadcrumb to every mystudy page
- change mystudy path/slug
- fix bug in quiz-review
- replace hexa code with tailwind config theme

### CODE REVIEW

**Comment by bangmar** [comment]
64 file gg gaming

**Comment by nandi-ir** [comment]
> 64 file gg gaming

awokwokwok

**Comment by maulanasdqn** [comment]
ngentot

File: apps/kg/modules/mystudy/assignment/index.tsx [hunk 1]
```diff
@@ -142,116 +141,114 @@ export const Status: FC = (): ReactElement => {
...
-                    {row.namaTabel}
-                  </div>
-                  <div
-                    className={`flex items-center col-span-3 py-[20px] px-[20px] border-solid border-b-[1px] border-[#D4D4D4] font-medium ${
-                      row.namaTabel === "Pengiriman Tugas" && "text-[#106FA4] flex-col gap-y-2"
-                    }
+    <section className="bg-white lg:py-[92px] md:py-[70px] py-[50px] lg:px-[109px] md:px-[70px] px-[50px] w-full">
+      <p className="text-[20px] font-semibold mb-[8px]">
+        {getMyStudyAssignmentItem?.assignment.title}
+      </p>
```
Comment (maulanasdqn) [inline apps/kg/modules/mystudy/assignment/index.tsx:152]: assigment?.title jaga jaga dia null or undefined

File: apps/kg/modules/mystudy/assignment/index.tsx [hunk 2]
```diff
@@ -142,116 +141,114 @@ export const Status: FC = (): ReactElement => {
...
+      <p className="text-[16px] font-medium mb-[8px]">
+        Tugas [Pertemuan ke-{getMyStudyAssignmentItem?.assignment.session_assignment.session_no}]
```
Comment (maulanasdqn) [inline apps/kg/modules/mystudy/assignment/index.tsx]: ini juga

File: apps/kg/modules/mystudy/assignment/index.tsx [hunk 3]
```diff
@@ -142,116 +141,114 @@ export const Status: FC = (): ReactElement => {
...
+      </p>
+      <p className="text-[16px] font-medium mb-[8px]">
+        {
+          getMyStudyAssignmentItem?.assignment.session_assignment.subject_session.teacher_subject
+            .full_name
+        }
+      </p>
+      <p className="text-[14px] mb-[35px] font-normal">
+        {getMyStudyAssignmentItem?.assignment.timestamp_created} WIB
+      </p>
```
Comment (maulanasdqn) [inline apps/kg/modules/mystudy/assignment/index.tsx:164]: ini juga

**Review by maulanasdqn — CHANGES_REQUESTED** [review]
cek cek sekua optional chaining mar

File: apps/kg/modules/mystudy/assignment/index.tsx (hunk 1 above)
Comment (nandi-ir) [reply apps/kg/modules/mystudy/assignment/index.tsx:152]: ok

**Comment by nandi-ir** [comment]
> ngentot

wkwk

**Review by maulanasdqn — APPROVED** [review]

**Pull request merged**